tools/makedeb/Makefile

```
@@ -2,10 +2,11 @@ LANG := en_US.UTF-8
 SHELL        := /usr/bin/env bash
 NAME         := makedeb
 VERSION      ?=
+ARCH  	     := $(shell [ -f "`which dpkg-architecture`" ]  && dpkg-architecture -qDEB_HOST_ARCH )
```

What happens if ``[ ! -f "`which dpkg-architecture`" ]``?  That probably shouldn't happen, so maybe throw a helpful error?

---

it quietly does nothing, and if you don't have dpkg-architecture, you do not have the ability to build debian packages anyways, as you are not on a debian-alike system.

---

> it quietly does nothing, and if you don't have dpkg-architecture, you do not have the ability to build debian packages anyways, as you are not on a debian-alike system.

then wouldn't it be better to just do this?:

```suggestion
ARCH  	     := $(shell dpkg-architecture -qDEB_HOST_ARCH )
```

i prefer "complain loudly" over "quietly do nothing" if what i clearly intend to do is not going to work.  (i'm talking about the general concept, in this case it's not that big a deal, so i already approved.)